lib/ab_test.rb

```diff
@@ -0,0 +1,96 @@
+class AbTest
+  extend Forwardable
+
+  attr_reader :allowed_variants, :expires
```

Comment:
Along with suggested changes to line 61

```suggestion
  attr_reader :allowed_variants, :expires, :frequency_sum
```